org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/ecore2xtext/Ecore2XtextExtensions.xtend

```diff
@@ -172,7 +172,9 @@ class Ecore2XtextExtensions {
 			'as',
 			'current',
 			'fragment',
-			'EOF'
+			'EOF',
+			'true',
+			'false'
```

Comment:
I see that isXtextKeyword should return true for `true`and `false`, but it doesn't really help given how this method is used.

Reply:
see org.eclipse.xtext.xtext.wizard.ecore2xtext.Ecore2XtextExtensions.quoteIfNeccesary(String)

Reply:
looks like we dont have a consistent concept for when to use ^ there

Reply:
(maybe for some backwards compatibility, so i would keep it for consistency

Reply:
so as we use validID everywhere we also could drop it here

Reply:
> so as we use validID everywhere we also could drop it here

:+1:

Reply:
done